README.md

```diff
@@ -142,8 +144,8 @@ For the flexible server (also see the [Azure Docs](https://docs.microsoft.com/en
 
 * You need an existing virtual network with a subnet
 * For the subnet enable the delegation to `Microsoft.DBforPostgreSQL/flexibleServers`
-* Create a private dns zone with a name that ends on `.postgres.database.azure.com` (e.g. `mydatabases.postgres.database.azure.com`)
-* Link the dns zone to your virtual network
+* Create a private dns zone with a name that ends on `.postgres.database.azure.com`, which can also be part of an other resource group (e.g. `mydatabases.postgres.database.azure.com`)
```

Comment:
Similar to line 145, i would phrase this as a prerequisite

Reply:
Changed in this commit
c1e4d7d35dd6b8bb60a3a357911932bc0f08759f